src/dbnode/server/limits.go

```diff
@@ -78,3 +83,75 @@ func validateProcessLimits() error {
 
 	return multiErr.FinalError()
 }
+
+func raiseRlimitToNROpen() error {
+	cmd := exec.Command("sysctl", "-a")
```

Comment:
Should just be able to check `/proc/sys`: https://www.kernel.org/doc/Documentation/sysctl/fs.txt

In kube:
```
/ # hostname
m3db-cluster-rep0-0
/ # cat /proc/sys/fs/nr_open
3000000
```

Reply:
Ah cool, yup I'll change it to this instead.